### PR TITLE
Story1 show collection

### PR DESCRIPTION
**Pull request by tomgarrett95**

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+function fetchDB(PDO $db): array {
+    $query = $db->prepare("SELECT `rides`.`name`, `rides`.`img`, `rides`.`height`, `rides`.`inversions`, `themeparks`.`park`
```
Comment (nealioa) [inline includes/functions.php:4]: The .sql file does not have a column called `park` in the table `themeparks`.

File: documents/rides.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  `inversions` int(11) unsigned DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_rides_locations` FOREIGN KEY (`location`) REFERENCES `themeparks`(`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=7 DEFAULT CHARSET=latin1;
+
+INSERT INTO `rides` (`id`, `name`, `location`, `img`, `height`, `inversions`) VALUES (1, 'Stealth', 1, 'stealth.jpg', 63, 0);
+INSERT INTO `rides` (`id`, `name`, `location`, `img`, `height`, `inversions`) VALUES (2, 'Colossus', 1, 'colossus.png', 30, 10);
+INSERT INTO `rides` (`id`, `name`, `location`, `img`, `height`, `inversions`) VALUES (3, 'Nemesis', 2, 'nemisis.jpg', 13, 4);
+INSERT INTO `rides` (`id`, `name`, `location`, `img`, `height`, `inversions`) VALUES (4, 'The Smiler', 2, 'the-smiler.jpg', 30, 14);
+INSERT INTO `rides` (`id`, `name`, `location`, `img`, `height`, `inversions`) VALUES (5, 'Black Mamba', 3, 'black-mamba.jpg', 26, 4);
```
Comment (nealioa) [inline documents/rides.sql]: The actual image file in the images folder is called `black-mamba.jpeg`.
Double-check the other names as well.

File: includes/tests/functions.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+        //assert
+        $this->assertEquals($expected, $result);
+    }
+
+    public function testDisplayCoasterMalformed() {
+        //arrange
+        $input = 746648.837;
+        $this->expectException(TypeError::class);
+        //act
```
Comment (nealioa) [inline includes/tests/functions.php:80]: Put a blank line above.

**Review by nealioa — CHANGES_REQUESTED** [review]

**Comment by tomgarrett95** [comment]
Changed SQL file.

**Review by nealioa — APPROVED** [review]

**Pull request merged**